shell/run_shell_cmd.go

```diff
@@ -22,5 +23,26 @@ func RunShellCommand(terragruntOptions *options.TerragruntOptions, command strin
 
 	cmd.Dir = terragruntOptions.WorkingDir
 
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, forwardSignals...)
+	defer signal.Stop(c)
+	defer close(c)
+
+	go func() {
+		for {
+			select {
+			case s := <-c:
+				if s == nil {
+					return
+				}
+				terragruntOptions.Logger.Printf("Forward signal %s to terraform.", s.String())
+				err := cmd.Process.Signal(s)
+				if err != nil {
+					terragruntOptions.Logger.Println(err)
+				}
+			}
+		}
+	}()
+
```

Comment:
Could you extract this code into a separate function with a clear name? Perhaps `forwardSignals`?